src/testing/downloads.fixtures.ts

```diff
@@ -0,0 +1,117 @@
+import { RouterModel, EpisodeModel, PodcastModel, ChartType, MetricsType,
+  CHARTTYPE_STACKED, INTERVAL_DAILY, METRICSTYPE_DOWNLOADS } from '../app/ngrx';
+import { MockHalDoc } from 'ngx-prx-styleguide';
+
+const podcast: PodcastModel = {
```

Comment:
Could probably be providing better fixture data, but this has needed to happen for a while.

Reply:
Oooh - nice.